lib/src/otx/data/transform_libs/torchvision.py

```diff
@@ -2040,7 +2036,7 @@ def forward(self, *_inputs: OTXDataItem) -> OTXDataItem | None:
         mixup_img = 0.5 * ori_img + 0.5 * padded_cropped_img.astype(np.float32)
 
         # TODO(ashwinvaidya17): remove this once we have a unified TorchDataItem
-        if isinstance(retrieve_results, OTXDataItem):
+        if isinstance(retrieve_results, (OTXDataItem, OTXSample)):
```

Comment:
I am leaving this for now. Once we have finished migrating the datasets to the new Datumaro API, I would like to simplify and unify the data structures used by the models.